x/dex/types/tick_iterator.go

```diff
@@ -1,5 +1,8 @@
 package types
 
 type TickIteratorI interface {
```

Comment:
Do we have a strong reason to put this in the types package? Seems more appropriate in the keeper package to me and reduces the need for these interfaces.

Reply:
More flexible to have things like this defined in the types package. Then they can be used by other types. If its in the keeper other types can't reference it.